pkg/registry/core/serviceaccount/strategy.go

```diff
@@ -18,15 +18,23 @@ package serviceaccount
 
 import (
 	"context"
+	"fmt"
 
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	"k8s.io/apiserver/pkg/storage/names"
+	"k8s.io/apiserver/pkg/warning"
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	api "k8s.io/kubernetes/pkg/apis/core"
 	"k8s.io/kubernetes/pkg/apis/core/validation"
 )
 
+const (
+	// EnforceMountableSecretsAnnotation (Deprecated) is a default annotation that indicates that a service account should enforce mountable secrets.
+	// The value must be true to have this annotation take effect
+	EnforceMountableSecretsAnnotation = "kubernetes.io/enforce-mountable-secrets"
```

Comment:
The use the `const` from `plugin/pkg/admission/serviceaccount` or unexport this one.